packages/owl-bot/src/owl-bot.ts

```diff
@@ -161,6 +182,14 @@ export function OwlBot(
         return;
       }
 
+      // Short-circuit if post-processor already running for this SHA:
+      const target = `${owner}_${repo}_${context.payload.pull_request.head.sha}`;
+      const lock = await owlbot.acquireLock(target);
+      if (!isLock(lock)) {
```

Comment:
If we just throw an exception, we won't have to add this clause either, just a try-catch block

Reply:
that's a reasonable suggestion 👍